hphp/runtime/ext/std/ext_std_options.php

```diff
@@ -295,7 +295,7 @@ function php_sapi_name(): string;
  * displaying the OS PHP was built on. This will only happen if your uname()
  * library call either doesn't exist or doesn't work.
  */
-<<__Native>>
+<<__Native, __IsFoldable>>
```

Comment:
I don't think this can be foldable. `php_uname('n')` can change are run time (if the hostname of the machine changes without restarting HHVM)